ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/433__data-science-pipelines-general.robot

```diff
@@ -0,0 +1,80 @@
+*** Settings ***
+Documentation       Test suite for OpenShift Pipeline API
+
+Resource            ../../../Resources/RHOSi.resource
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Common.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataSciencePipelines.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource            ../../../Resources/Page/Operators/OpenShiftPipelines.resource
+Library             DateTime
+Library             ../../../../libs/DataSciencePipelinesAPI.py
+Suite Setup         General Suite Setup
+Suite Teardown      General Suite Teardown
+
+
+*** Variables ***
+${PROJECT_USER2}=    dspa-project-user2
+${PROJECT_USER3}=    dspa-project-user3
+${S3_BUCKET}=    ods-ci-ds-pipelines
+
+
+*** Test Cases ***
+Verify Ods User Can Bind The Route Role
+    [Documentation]    Verify Ods User Can Bind The Route Role
```

Comment:
could provide a slightly more detailed documentation to understand what the TC does?

Reply:
Done